### ISSUE TITLE
Override configuration

### ISSUE THREAD

**Issue by BarAluf** [devopshq/teamcity#18]
Hi,
i have an ssl error which i can solve by insert my own certificate to your API client,
but i don't find a way to override your configuration file.
[Conf file](https://github.com/devopshq/teamcity/blob/develop/dohq_teamcity/configuration.py)
thanks, 
Bar Aluf

**Comment by allburov** [comment]
@BarAluf yep, do not exist correct way pass some parameters to configuration right now, unfortunately.
I fixed this in `develop`-branch.

Could you try this give me feedback? I will add docs and merge to master this If it's solve you problem

Install package from git
```bash
pip install git+https://github.com/devopshq/teamcity#develop
```

```python
conf = Configuration()
conf.ssl_ca_cert = '/path/to/file.crt'
# OR quick hack for testing
# conf.verify_ssl = False
tc = dohq_teamcity.TeamCity("http://teamcity.example.com", (sys.argv[1], sys.argv[2]), configuration=conf)
````

**Comment by allburov** [comment]
Solution available in version `1.0.2` in pypi. But I forgot to write an example in the documentation. I will do as I find free time
